docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -61,7 +57,7 @@ SOURCE_GALERA_MEMBERS=(
 SOURCE_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
 ----
 
-* Prepare MariaDB copy directory and the adoption helper pod
+* Prepare MariaDB copy directory and the adoption helper pod 
 
 * Create a temporary folder to store adoption helper pod (pick storage requests to fit MySQL database size):
```

Comment:
In the preview, the yaml file content with the apiVersion stuff needs to be indented. Not part of this PR.